feat(HTTP): Refresh 4XX response pages (#34620)
mdn/content: files/en-us/web/http/status/409/index.md

{{HTTPSidebar}}

The HTTP **`409 Conflict`** status response code is part of the `400`-`499` class of [client error responses](/en-US/docs/Web/HTTP/Status#client_error_responses) and indicates a request conflict with the current state of the target resource.

> In WebDAV, 409 Conflict responses are errors sent to the client so that a user might be able to resolve a conflict and resubmit the request.

Reviewer: ```suggestion
In {{glossary("WebDAV")}} remote web authoring, 409 conflict responses are errors sent to the client so that a user might be able to resolve a conflict and resubmit the request.
```

Author: done in 5ec35dbdd45b72e904277ceab6f6106a745a6c10

For example, conflicts occur if a request to create collection `/a/b/c/d/` is made, and `/a/b/c/` does not exist, the request must fail with a 409.
Additionally, you may get a 409 response when uploading a file that is older than the existing one on the server, resulting in a version control conflict.

In other systems, 409 responses may be used for implementation-specific purposes, such as to indicate that the server has received multiple requests to update the same resource.

## Status

```http
409 Conflict
```

## Examples

### Concurrent tasks disallowed

In the following example, we want to kick off an automation process that performs a common task in the system:

```http
POST /tasks HTTP/1.1
Host: example.com
Content-Type: application/json

{
  "task": "emailDogOwners",
  "template": "pickup"
}
```

> In this implementation, the server disallows two concurrent jobs from running and returns a 409 so the client can check if they meant to perform the action or run a different task:

Reviewer: ```suggestion
In this implementation, the server disallows two concurrent jobs from running and returns a 409, providing the client an opportunity to check if they meant to perform the action or run a different task:
```

Author: done in https://github.com/mdn/content/pull/34620/commits/5ec35dbdd45b72e904277ceab6f6106a745a6c10#diff-8d39af0ed97d2304df9faf82f652638ced649433467d1482d262ee22b6ce3053R42
